packages/create-slinkity/template/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A splendid starter for my Slinkity site!",
   "main": ".eleventy.js",
   "scripts": {
-    "start": "slinkity --serve --incremental",
+    "dev": "slinkity --serve --incremental",
```

Comment:
What if we did something like:

```json
{
  "scripts": {
    "dev": "slinkity --serve --incremental",
    "start": "slinkity --serve --incremental",
    "build": "slinkity"
  }
}
```

So the users could just do whichever they're used to (either `npm start` or `npm run dev`) and will get essentially the same result.

This still leaves `serve` open if at some point we may want to differentiate between a `dev` command and a `serve` command.

Reply:
Sure, this sounds good to me!